### PR TITLE
Inject to frames functionality

### PR DESCRIPTION
**Pull request by jdmesalosada**

### CODE REVIEW

File: javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using javnov.Selenium.Axe;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium;
+using FluentAssertions;
+
+namespace AxeCsharpSeleniumTest
+{
+    [TestClass]
+    public class UnitTest1
```
Comment (javnov) [inline javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs]: Rename class and file name

File: javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        [TestCleanup()]
+        public virtual void TearDown()
+        {
+            _webDriver.Quit();
+            _webDriver.Dispose();
+        }
+
+
+        [TestMethod]
+        public void TestMethod1()
```
Comment (javnov) [inline javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs]: put a more meaningful method name

File: javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs [hunk 3]
```diff
@@ -27,6 +29,8 @@ public class AxeBuilder
 
             _webDriver = webDriver;
             _webDriver.Inject();
+            Options = "null";
```
Comment (javnov) [inline javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs]: why this is necessary here ?

File: javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs [hunk 4]
```diff
@@ -114,14 +129,8 @@ public JObject Analyze()
...
 
-            return Execute(command);
-        }
-
-        private JObject Execute(string command, params object[] args)
-        {
-            _webDriver.Manage().Timeouts().SetScriptTimeout(TimeSpan.FromSeconds(30));
-            object response = ((IJavaScriptExecutor)_webDriver).ExecuteAsyncScript(command, args);
-            return new JObject(response);
+            var violations = Execute(command).SelectToken("violations");
```
Comment (javnov) [inline javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs]: Split in two lines, like this:
```csharp
JObject executionResult = Execute(command);
var violations = executionResult.SelectTokect("violations")
```

Also **stop using var for unknown types**, in this case we don't know which **violations** variable type is just reading the code.

**Review by javnov — CHANGES_REQUESTED** [review]

File: javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs (hunk 1 above)
Comment (jdmesalosada) [reply javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs]: Done.

File: javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs (hunk 2 above)
Comment (jdmesalosada) [reply javnov.Selenium.Axe/AxeCsharpSeleniumTest/UnitTest1.cs]: Done.

File: javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs (hunk 3 above)
Comment (jdmesalosada) [reply javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs]: No. Done.

**Review by javnov — APPROVED** [review]

**Pull request merged**